app.rb

```diff
@@ -0,0 +1,79 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'nameable'
+require_relative 'trimmer_decorator'
+require_relative 'capitalize_decorator'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    @books.each do |book|
+      puts "#{book.title} by #{book.author}"
+    end
+  end
```

Comment:
- [ ] When displaying the book lists you should display their indexes to be able to use them while creating a rental

Reply:
resolved thanks